### ISSUE TITLE
Documentation links are broken

### ISSUE THREAD

**Issue by c0deaddict** [immutable-js/immutable-js#1443]
The links in the online documentation (https://facebook.github.io/immutable-js/docs/) are broken. When clicking on "List" for example, I get redirected to https://facebook.github.io/List

**Comment by timothyjlaurent** [comment]
I'm seeing this too... DevDocs to the rescue!!

**Comment by AnilRedshift** [comment]
This makes it very hard to use immutable when you can't even see what functions are available. @flarnie: As the last contributor to the repo can you have someone take a look at this

**Comment by AnilRedshift** [comment]
For anyone who sees this and needs access to the docs, I pulled in https://github.com/facebook/immutable-js/pull/1444 and uploaded the files here: https://immutable-docs.herokuapp.com/docs/

**Comment by flarnie** [comment]
Hi - I honestly don't know how the docs for Immutable.js are built and updated. 

Also when I visit the docs the links are working for me. Tips on reproducing this issue?

@leebyron do you have ideas about this? And for the future, can you point me at any docs about the docs? 

![ilovedocs2](https://user-images.githubusercontent.com/1114467/37235470-c6d15506-23b3-11e8-98b0-66a44f2ff5b1.gif)

**Comment by AnilRedshift** [comment]
Thanks @flarnie for taking a look.

TL;DR: Looks like a falseish alarm.

I was still able to repro, but then on a whim decided to try on incognito mode... and it started working. I figured out that one of extensions blocks jsdelivr.net (which is loading the react runtime).

However, instead of breaking in any of the usual ways, it does the whole redirect to https://facebook.github.io/List instead of https://facebook.github.io/#List.

Allowing that call through has the docs working again.

So, there _is_ still a bug in that the HTML that is generated without javascript has the incorrect links. However, even if you manually put in the correct URL, the docs don't load because the website is really a react app. If I were you folks I would either close this as a won't-fix, or add some kind of <noscript> warning if loading react from the CDN fails. OR move the javascript file to a 1st party URL so it's not blocked by default.

**Comment by jdeniau** [comment]
I think all doc links should work since 4.0.0 and the new website.
